lib/engine/part/halt.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative 'town'
+
+module Engine
+  module Part
+    class Halt < Town
+      attr_reader :symbol
+
+      def initialize(symbol, **opts)
+        @revenue = parse_revenue('0', opts[:format])
```

Comment:
seems luke duplicate code with town, can you call super and override?

Reply:
Ok, need to learn how to change arguments to the super